default/scripting/species/common/industry.macros

```diff
@@ -5,15 +5,25 @@ BASIC_INDUSTRY
 '''        EffectsGroup
             scope = Source
             activation = And [
-                Source
                 Planet
                 TargetIndustry low = 0
                 Focus type = "FOCUS_INDUSTRY"
             ]
             accountinglabel = "FOCUS_INDUSTRY_LABEL"
-            priority = [[EARLY_PRIORITY]]
+            priority = [[DEFAULT_PRIORITY]]
```

Comment:
Then why does the equivalent effect for research use default priority?

https://github.com/freeorion/freeorion/blob/master/default/scripting/species/common/research.macros#L10